docs/yourbase-ruby.md

```diff
@@ -0,0 +1,73 @@
+# YourBase Ruby Acceleration
+
+Tests are important. For large monoliths, they're also a major source of drag on velocity.
+
+YourBase is a tool that traces your tests to determine which functions each test depends on. It can later use this information to determine which tests do not need to run because their code paths have not changed. These tests are skipped automatically.
+
+YourBase works with Ruby versions >= 2.3
+
+## Getting Started
+No configuration, setup, or babysitting required. To get started, you need the YourBase gem and a download token.  To request a token, please visit [YourBase.io](https://yourbase.io/download)
+
+Once you have a token, simply follow the steps below:
+```sh
+bundle add yourbase-rspec --git "https://${YOURBASE_DOWNLOAD_TOKEN?}:x-oauth-basic@github.com/yourbase/yourbase-rspec-skipper-engine.git" && bundle install
+```
+
+## First run
+After installing yourbase-rspec, if you are not using Rails you must add
+"require 'yourbase-rspec'" in your spec folder.
+
+```sh
+# Add require for non Rails projects
+echo "require 'yourbase-rspec'" >> spec/yourbase_spec.rb
```

Comment:
Let's chat.  I wanted to give an example of a simple, quick way to add in our requirement without touching their files.

Reply:
we can chat at/after standup, but they already have to change a project file to add the gem. And looks like what this will do is, assuming they are at the root dir of the project, add a file named yourbase_spec.rb to the spec folder that only contains the require line? I don't think that will get picked up if they run a specific spec file from their project.

Reply:
I think this `echo` line needs to be removed before we publish this. It will stop a nontrival percentage of rails devs in their tracks as incomprehensible or exceptionally odd.

```suggestion
If you are not in a Rails project, you will also need to add `require 'yourbase-rspec'` to your `spec_helper.rb` (or another project file which will be loaded, such as the file containing the specs you are running).
```